examples/bulk_api_usecases/bulk-operations-csv/bulk_delete_csv.bal

```diff
@@ -28,7 +28,7 @@ configurable string refreshUrl = os:getEnv("REFRESH_URL");
 configurable string baseUrl = os:getEnv("EP_URL");
 
 // Using direct-token config for client configuration
-salesforce:ConnectionConfig sfConfig = {
+bulk:ConnectionConfig sfConfig = {
```

Comment:
Is there a reason for using a new type here?

Reply:
Yes. Salesforce module has defined ConnectionConfig object in all three types of APIs Rest, SOAP as well as bulk. Since we are using the bulk client, this should be changed to bulk: ConnectionConfig. Otherwise it will give errors as we have added multiple new auth flows to the previously used REST ConnectionConfig